### ISSUE TITLE
HS110 Plug accessory created for each homebridge instance

### ISSUE THREAD

**Issue by sebiprinz** [plasticrake/homebridge-tplink-smarthome#103]
I have 9 seperate homebridge instances running (one for each plugin). The reason why, is that in case one plugin is not working correctly, my whole homebridge setup won't completely stop working. The problem is that the homebridge-tplink-smarthome plugin is creating accessories for each homebridge instance, even though i have only added it to _only one_ config.json. The result is that I have 9 duplicate HS110 plugs in Homekit.

### Expected Behavior
Only create accessories when the platform is added to the config.json. Do not create accessories when the platform is not added to the config.json.

### Current Behavior
Accessories are created for each homebridge instance, even though the platform is not added to the config.json. In my case, there are 9 duplicate plugs in Homekit (one for each homebridge instance) even though I have added the platform to only one config.json/homebridge instance.

### Steps to Reproduce (for bugs)
Create two or more homebridge instances. Install plugin. Specify platform in just one config.json. Restart both homebridge instances. Accessories are created multiple times, in each homebridge instance.

### Versions
node -vversion: v12.10.0
homebridge --version: 0.4.50
OS: Raspberry Pi 3 B+ running Raspbian GNU/Linux 9.11

### Configuration
```json
{
    "bridge": {
        "name": "Homebridge8",
        "username": "CC:22:3D:E3:CE:48",
        "port": 51844,
        "pin": "031-45-172"
    },
    "description": "TP-Link Plugin",
    "platforms": [
        {
	"platform": "TplinkSmarthome",
	"name": "TplinkSmarthome",
	"pollingInterval": 30,
	"addCustomCharacteristics": true,
	"inUseThreshold": 4,
	"timeout": 15
	}
     ],
    "accessories": []
}
```

### Homebridge Log / Command Output
Output for homebridge instance "homebridge8" with configured config.json file
```
Dez 22 21:14:34 raspberrypi systemd[1]: Started Node.js HomeKit Server.
Dez 22 21:14:35 raspberrypi homebridge[5514]: [2019-12-22 21:14:35] Loaded config.json with 0 accessories and 1 platforms.

loading log for other plugins - irrelevant

Dez 22 21:14:40 raspberrypi homebridge[5514]: [2019-12-22 21:14:40] ---
Dez 22 21:14:40 raspberrypi homebridge[5514]: [2019-12-22 21:14:40] Loaded plugin: homebridge-tplink-smarthome
Dez 22 21:14:40 raspberrypi homebridge[5514]: [2019-12-22 21:14:40] Registering platform 'homebridge-tplink-smarthome.TplinkSmarthome'
Dez 22 21:14:40 raspberrypi homebridge[5514]: [2019-12-22 21:14:40] ---

loading log for other plugins - irrelevant

Dez 22 21:14:42 raspberrypi homebridge[5514]: [2019-12-22 21:14:42] ---
Dez 22 21:14:42 raspberrypi homebridge[5514]: [2019-12-22 21:14:42] Loading 1 platforms...
Dez 22 21:14:42 raspberrypi homebridge[5514]: [2019-12-22 21:14:42] [TplinkSmarthome] Initializing TplinkSmarthome platform...
Dez 22 21:14:42 raspberrypi homebridge[5514]: [2019-12-22 21:14:42] [TplinkSmarthome] homebridge-tplink-smarthome v4.0.1, node v12.10.0, homebridge v0.4.50
Dez 22 21:14:42 raspberrypi homebridge[5514]: [2019-12-22 21:14:42] Loading 0 accessories...
Dez 22 21:14:42 raspberrypi homebridge[5514]: Load homebridge-harmonyHub.HarmonyHubWebSocket
Dez 22 21:14:42 raspberrypi homebridge[5514]: [2019-12-22 21:14:42] [homebridge-harmonyHub.HarmonyHubWebSocket] HarmonyPlatform Init
Dez 22 21:14:42 raspberrypi homebridge[5514]: [2019-12-22 21:14:42] [homebridge-harmonyHub.HarmonyHubWebSocket] No configuration found for homebridge-harmonyHub
Dez 22 21:14:42 raspberrypi homebridge[5514]: Load homebridge-telegram.Telegram
Dez 22 21:14:42 raspberrypi homebridge[5514]: [2019-12-22 21:14:42] [homebridge-telegram.Telegram] TelegramPlatform Plugin Loaded
Dez 22 21:14:42 raspberrypi homebridge[5514]: [2019-12-22 21:14:42] [TplinkSmarthome] Configuring cached accessory: [Waschmaschine] 80062EEE1A3FEBBD966A83ECB47A41321B899F04 426ba26e-2a37-49de-bb04-f11a42a9e89e
Dez 22 21:14:42 raspberrypi homebridge[5514]: Setup Payload:
Dez 22 21:14:42 raspberrypi homebridge[5514]: X-HM://0023ISYXGEMDE
Dez 22 21:14:42 raspberrypi homebridge[5514]: Scan this code with your HomeKit app on your iOS device to pair with Homebridge:

QR code - irrelevant
                                                  
Dez 22 21:14:42 raspberrypi homebridge[5514]:                                                       
Dez 22 21:14:42 raspberrypi homebridge[5514]:                                                       
Dez 22 21:14:42 raspberrypi homebridge[5514]: Or enter this code with your HomeKit app on your iOS device to pair with Homebridge:
Dez 22 21:14:42 raspberrypi homebridge[5514]:                        
Dez 22 21:14:42 raspberrypi homebridge[5514]:     ┌────────────┐
Dez 22 21:14:42 raspberrypi homebridge[5514]:     │ 031-45-172 │
Dez 22 21:14:42 raspberrypi homebridge[5514]:     └────────────┘
Dez 22 21:14:42 raspberrypi homebridge[5514]:                        
Dez 22 21:14:42 raspberrypi homebridge[5514]: [2019-12-22 21:14:42] Homebridge is running on port 51844.
Dez 22 21:15:12 raspberrypi homebridge[5514]: [2019-12-22 21:15:12] [TplinkSmarthome] New Device Online: [Waschmaschine] plug [80062EEE1A3FEBBD966A83ECB47A41321B899F04] 192.168.0.45 9999
Dez 22 21:15:12 raspberrypi homebridge[5514]: [2019-12-22 21:15:12] [TplinkSmarthome] Adding: [Waschmaschine] plug [80062EEE1A3FEBBD966A83ECB47A41321B899F04]
```
The device discovery (see log above) happens for every homebridge instance.

**Comment by roberkane212** [comment]
I have the same issue

**Comment by daanwa** [comment]
Same issue here. Very annoying.

**Comment by sebiprinz** [comment]
Issue was fixed with new homebridge version 1.0.0+
